### ISSUE TITLE
How do I generate the database tables?

### ISSUE THREAD

**Issue by jesperstarkar** [jaskie/PlayoutAutomation#4]
I can't find any way of installing the database tables needed. As I try to config I see what's missing for each error message appearing, but I can't think this is the way to reverse-engineer the database structure.

**Comment by jesperstarkar** [comment]
I found this one now: https://github.com/jaskie/PlayoutAutomation/blob/fa67dff52c267b2bd96e61d95d00477de51c0b21/TAS.Server.Common/Database/database.sql#L122

**Comment by jaskie** [comment]
Use the TVPlay.config program, button _Config file and database_-> at the bottom part of the window a buttons for it.
![configfile](https://cloud.githubusercontent.com/assets/1919742/15041267/fe044bca-12ba-11e6-9fb0-474bf066dcf8.png)

**Comment by jesperstarkar** [comment]
Got it! 

However, doing so, I guess it runs the .sql-file I linked to above. And it throws errors. In general i don't like time(6) or timestamp(3), timestamp(6), so something about the lenght param is invalid syntax. After stripping those manually I got an error on setting key "KEY `idPlaystate` (`PlayState`) USING BTREE".

Any ideas why the SQL throws errors? I have a fresh install of MAMP and have sucessfully connected to the database.
